Squablr/Views/CardView.m

```diff
@@ -25,6 +25,89 @@ - (instancetype)initWithFrame:(CGRect)frame {
+    self = [super initWithFrame:frame];
+    if (self) {
+        [self setup];
+        self.profile = profile;
+        // Creating UIImage view programmatically
+        PFImageView *imageView = [[PFImageView alloc] initWithFrame:CGRectMake(0, 0, 350, 500)];
+        imageView.image = nil;
+        imageView.file = [profile[@"profileImages"] firstObject]; // TODO: Change this to the user profile images
+        [imageView loadInBackground];
+        imageView.contentMode = UIViewContentModeScaleAspectFill;
+        imageView.layer.cornerRadius = 10.0;
+        imageView.layer.masksToBounds = true;
+        [self addSubview:imageView];
+        // Name Label
+        UILabel *name = [[UILabel alloc]initWithFrame:CGRectMake(25, 500, 200, 40)];
+        name.text = profile.name;
+        name.numberOfLines = 0;
+        name.baselineAdjustment = UIBaselineAdjustmentAlignBaselines; // or UIBaselineAdjustmentAlignCenters, or UIBaselineAdjustmentNone
+        name.adjustsFontSizeToFitWidth = YES;
+        name.minimumScaleFactor = 10.0f/12.0f;
+        name.clipsToBounds = YES;
+        name.backgroundColor = [UIColor clearColor];
+        name.textColor = [UIColor blackColor];
+        name.textAlignment = NSTextAlignmentLeft;
+        name.translatesAutoresizingMaskIntoConstraints = NO;
+        [self addSubview:name];
+        // Age Label
+        UILabel *age = [[UILabel alloc]initWithFrame:CGRectMake(125, 500, 200, 40)];
+        age.text = [profile.age stringValue];
+        age.numberOfLines = 0;
+        age.baselineAdjustment = UIBaselineAdjustmentAlignBaselines; // or UIBaselineAdjustmentAlignCenters, or UIBaselineAdjustmentNone
+        age.adjustsFontSizeToFitWidth = YES;
+        age.minimumScaleFactor = 10.0f/12.0f;
+        age.clipsToBounds = YES;
+        age.backgroundColor = [UIColor clearColor];
+        age.textColor = [UIColor blackColor];
+        age.textAlignment = NSTextAlignmentLeft;
+        age.translatesAutoresizingMaskIntoConstraints = NO;
+        [self addSubview:age];
+        // Biography label
+        UILabel *bio = [[UILabel alloc]initWithFrame:CGRectMake(25, 540, 200, 40)];
+        bio.text = profile.biography;
+        bio.numberOfLines = 0;
+        bio.baselineAdjustment = UIBaselineAdjustmentAlignBaselines; // or UIBaselineAdjustmentAlignCenters, or UIBaselineAdjustmentNone
+        bio.adjustsFontSizeToFitWidth = YES;
+        bio.minimumScaleFactor = 10.0f/12.0f;
+        bio.clipsToBounds = YES;
+        bio.backgroundColor = [UIColor clearColor];
+        bio.textColor = [UIColor blackColor];
+        bio.textAlignment = NSTextAlignmentLeft;
+        bio.translatesAutoresizingMaskIntoConstraints = NO;
+        [self addSubview:bio];
+        // Name label layout
+        NSLayoutConstraint *left = [NSLayoutConstraint constraintWithItem:name attribute:NSLayoutAttributeLeft relatedBy:NSLayoutRelationEqual toItem:self attribute:NSLayoutAttributeLeft multiplier:1 constant:10];
```

Comment:
Be more specific with the naming here, like `nameLeftEdgeToParent` or something, same with the others below